integration_test/src/main.rs

```diff
@@ -998,6 +999,10 @@ fn test_get_tx_out_set_info(cl: &Client) {
     cl.get_tx_out_set_info().unwrap();
 }
 
+fn test_get_chain_tips(cl: &Client) {
+    cl.get_chain_tips().unwrap();
```

Comment:
Maybe we can at least assert `len()==1`? Would be nice if we knew the last generated block and could check against that, but that's probably a quite invasive change to the integration tests.

Reply:
Happy to add a that if there is more interest. Was following [this](https://github.com/rust-bitcoin/rust-bitcoincore-rpc/blob/7048492e398c267d37f841a0eb31240aea741b18/integration_test/src/main.rs#L7-L8) comment to only test the serialization and not the correctness of the server. Length assert does make sense to test that there is actually something being serialized.

Reply:
> Length assert does make sense to test that there is actually something being serialized.

Exactly, that was the main intention behind it. I think it would be nice to test more (e.g. some examples with chain splits to detect changes in the json schema in that case in the future, which isn't really "testing the server" imo, but that would be a lot more work to do consistently and is probably not worth it currently.

Reply:
It's possible (and easy) to get a `GetChainTipsResultStatus::ValidFork` tip wiht `invalidateblock` followed by a `generate`. Not too sure about `::HeadersOnly`, `::ValidHeaders` or even `::Invalid`.